virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -58,21 +58,33 @@ endif::openshift-rosa,openshift-dedicated[]
 [id="secondary-network-config"]
 == Configuring VM secondary network interfaces
 
-You can connect a virtual machine to a secondary network by using Linux bridge, SR-IOV and OVN-Kubernetes CNI plugins. You can list multiple secondary networks and interfaces in the VM specification. It is not required to specify the primary pod network in the VM specification when connecting to a secondary network interface.
+You can connect a virtual machine to a secondary network by using OVN-Kubernetes, SR-IOV, and Linux bridge CNI plugins. You can list multiple secondary networks and interfaces in the VM specification. It is not required to specify the primary pod network in the VM specification when connecting to a secondary network interface.
 
-// Hiding from ROSA/OSD as Linux Bridge not supported
-ifndef::openshift-rosa,openshift-dedicated[]
-xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[Connecting a virtual machine to a Linux bridge network]::
+xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[Connecting a virtual machine to an OVN-Kubernetes secondary network]::
 
-xref:../../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Install the Kubernetes NMState Operator] to configure Linux bridges, VLANs, and bondings for your secondary networks.
+You can connect a VM to an Open Virtual Network (OVN)-Kubernetes secondary network. {VirtProductName} supports the layer 2 and localnet topologies for OVN-Kubernetes. The localnet topology is the recommended way of exposing VMs to the underlying physical network, with or without VLAN encapsulation.
```

Comment:
I would not expand the short in this case - it is a name of the project, not a short for the name of the project.

Reply:
plus, I'd keep `layer2` without the space. It's the configuration value for the `topology` attribute. As is localnet. 

Does it make sense ?

Reply:
Agreed

Reply:
@phoracek Do you mean not spell out Open Virtual Network (OVN)?

Reply:
The change from `layer 2` to `layer2` is a wider change affecting multiple modules. Might be easier to create a separate bug for that?

Reply:
fine by me

Reply:
@sjhala-ccs yes, I would only spell "OVN-Kubernetes" everywhere